pkg/server/plugin/upstreamauthority/awspca/pca_test.go

```diff
@@ -226,17 +234,21 @@ func (as *PCAPluginSuite) Test_MintX509CA_IssuanceWaitError() {
 	// But the wait call returns an error
 	as.verifyWaitUntilCertificateIssued(errors.New("issuance waiting error"))
 
-	// The resulting response should error
-	response, err := as.plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{
+	stream, err := as.plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{
 		Csr:          csr,
 		PreferredTtl: testTTL,
 	})
+	as.Require().NoError(err)
+	as.Require().NotNil(stream)
+
+	// The resulting response should error
+	response, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns `io.EOF`.